### PR TITLE
chore: bump version to beta.8 w/ changelog

### PR DESCRIPTION
**Pull request by jelbourn**
cc @DevVersion the version script didn't update the CDK versions where necessary so I temporarily hard-coded them

### CODE REVIEW

**Comment by DevVersion** [comment]
@jelbourn Yeah this is probably because #4962 is not merged yet. I mentioned it when we started moving stuff to the CDK: https://github.com/angular/material2/pull/5025#issue-234561191

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+initial release, code from Angular Material's `core/` have been moved for `a11y/`, `bidi/`,
+`coercion/`, `observe-content/`, `platform/`, `portal/`. The `@angular/material` package now
+re-exports these symbols, marked as deprecated. The re-exports will be removed in a subsequent
+release.
+* Initial version of data-table component. There is both a `<cdk-table>` (the core) and the
+`<md-table>` (with Material Design styles). See the documentation on material.angular.io for more
+information.
+* Initial version of `<md-paginator>` and `<md-sort-header>` components, which can be used either
+with `<md-table>` or any other table.
+* Both `@angular/material` and `@angular/cdk` are now strict null complicant.
```
Comment (crisbeto) [inline CHANGELOG.md]: Shouldn't this one be "compliant"?

Comment (jelbourn) [inline CHANGELOG.md]: Fixed

File: src/cdk/package.json [hunk 2]
```diff
@@ -1,12 +1,11 @@
 {
   "name": "@angular/cdk",
-  "version": "0.0.0-PLACEHOLDER",
+  "version": "2.0.0-beta.8",
```
Comment (DevVersion) [inline src/cdk/package.json]: Are you sure about this one? The only thing that's reasonable (why it not updated the placeholder) is actually the peer dependency from Material.

Comment (jelbourn) [inline src/cdk/package.json]: I may have been overzealous, but we can switch it back later today.

Comment (DevVersion) [inline src/cdk/package.json]: Fine by me.

**Pull request merged**